### DISCUSSION TITLE
How to use a Resource in multiple panels (with different level of controls) ?

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [filamentphp/filament#12126] [Help]
### Package

Panel builder

### Package Version

v3.2.61

### How can we help you?

Hi,
I want to access a resource in multiple panels with different level of controls. For ex: PostResource, which should be available in both AdminPanel, and ManagerPanel. So this question has two parts.

1. How do I link the PostResource to these two panels. I understand when creating resource, it asks for a Panel, in which it works fine. How can I show it in another panel ?

2. How to offer different level of controls to user of these two Panels. Example- Manager should not be able to delete the posts. 

Thanks

**Comment by piyushdongre-dot-com** [comment]
I found solution for these.
1. I realised, we can create resources with different names, for same model, and attach it will different panels(while creating)
2. We can modify getEloquentQuery() in the respective resources.

**Reply by leandrocfe** [reply]
You could create a class and extend the resource:

`app/Filament/Manager/Resources/PostResource.php`

```php
namespace App\Filament\Manager\Resources;

//PostResource from the admin panel
use App\Filament\Admin\Resources\PostResource as BasePostResource;

class PostResource extends BasePostResource
{
    //override what you need..
}
```
